apps/web/src/components/NotificationList/NotificationListPage.tsx

```diff
@@ -0,0 +1,121 @@
+import React from "react";
+import { useIntl } from "react-intl";
+import { useQuery } from "urql";
+import { useSearchParams } from "react-router-dom";
+
+import { graphql } from "@gc-digital-talent/graphql";
+import { unpackMaybes } from "@gc-digital-talent/helpers";
+import { Link, Pending, Well } from "@gc-digital-talent/ui";
+
+import NotificationItem from "./NotificationItem";
+
+const Notifications_Query = graphql(/* GraphQL */ `
+  query Notifications(
+    $where: NotificationFilterInput
+    $first: Int
+    $page: Int
+  ) {
+    notifications(where: $where, first: $first, page: $page) {
+      paginatorInfo {
+        hasMorePages
+      }
+      data {
+        id
+        ...NotificationItem
+      }
+    }
+  }
+`);
+
+const PER_PAGE = 10;
+
+interface NotificationPageProps {
+  page: number;
+  onlyUnread?: boolean;
+  isLastPage?: boolean;
+}
+
+const NotificationListPage = ({
+  page,
+  onlyUnread,
+  isLastPage,
+}: NotificationPageProps) => {
+  const intl = useIntl();
+  const [searchParams] = useSearchParams();
+  searchParams.set("page", `${page + 1}`);
+  const [{ data, fetching, error }] = useQuery({
+    query: Notifications_Query,
+    variables: {
```

Comment:
Should there be a sorting command here?  To ensure pages are read out coherently?

Reply:
Oh, I missed that.  No, this is good, at least for now.